src/Vlingo.Xoom.Streams.Tests/StreamSubscriberTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using Vlingo.Xoom.Streams.Sink.Test;
+using Xunit;
+
+namespace Vlingo.Xoom.Streams.Tests
+{
+  public class StreamSubscriberTest : StreamPubSubTest
+  {
+    private readonly SafeConsumerSink<string> _sink;
+
+    public StreamSubscriberTest()
+    {
+      _sink = new SafeConsumerSink<string>();
+    }
+
+    [Fact]
+    public void TestThatSubscriberSubscribes()
+    {
+      // GIVEN
+      CreatePublisherWith(SourceOfABC);
+      CreateSubscriberWith(_sink, 2);
+
+      var access = _sink.AfterCompleting(1);
+
+      // WHEN
+      Publisher.Subscribe(Subscriber);
+
+      // THEN
+      var subscriberCount = access.ReadFrom<int>("ready");
+      Assert.Equal(1, subscriberCount);
+    }
+
+    [Fact(Skip = "WIP")]
```

Comment:
Likewise, when you wanted to unskip it ?

Reply:
Yes, will be fixed in the next PR..